docs/source/en/model_doc/dpt.mdx

```diff
@@ -28,37 +28,42 @@ alt="drawing" width="600"/>
 
 This model was contributed by [nielsr](https://huggingface.co/nielsr). The original code can be found [here](https://github.com/isl-org/DPT).
 
+## Resources
+
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with DPT.
+
+<PipelineTag pipeline="depth-estimation"/>
```

Comment:
This is what likely causes the build failure. The depth estimation pipeline. 
When trying to build this page locally I'm getting :
500
Cannot read properties of undefined (reading 'name')
TypeError: Cannot read properties of undefined (reading 'name')
    at PipelineTag.svelte:12:31
    at Object.$$render (/private/var/folders/jb/59bqr82j2mx1qv1zqbt4ntn80000gn/T/tmpuwohzafj/kit/node_modules/svelte/internal/index.js:1745:22)
    at eval (/src/routes/model_doc/glpn.mdx:126:91)
    at Object.$$render (/private/var/folders/jb/59bqr82j2mx1qv1zqbt4ntn80000gn/T/tmpuwohzafj/kit/node_modules/svelte/internal/index.js:1745:22)
    at Object.default (root.svelte:43:47)
    at eval (/src/routes/__layout.svelte:40:157)
    at Object.$$render (/private/var/folders/jb/59bqr82j2mx1qv1zqbt4ntn80000gn/T/tmpuwohzafj/kit/node_modules/svelte/internal/index.js:1745:22)
    at root.svelte:37:45
    at $$render (/private/var/folders/jb/59bqr82j2mx1qv1zqbt4ntn80000gn/T/tmpuwohzafj/kit/node_modules/svelte/internal/index.js:1745:22)
    at Object.render (/private/var/folders/jb/59bqr82j2mx1qv1zqbt4ntn80000gn/T/tmpuwohzafj/kit/node_modules/svelte/internal/index.js:1753:26)

Removing the pipeline tag solves the issue.